packages/esm-patient-registration-app/src/config-schema.ts

```diff
@@ -199,6 +205,16 @@ export const esmPatientRegistrationSchema = {
       'Definitions for custom fields that can be used in sectionDefinitions. Can also be used to override built-in fields.',
   },
   fieldConfigurations: {
+    causeOfDeath: {
+      conceptUuid: {
+        _type: Type.String,
+        _description: 'The concept UUID to get cause of death answers',
+        _default: '9272a14b-7260-4353-9e5b-5787b5dead9d',
+      },
+      validation: {
+        required: { _type: Type.Boolean, _default: false },
```

Comment:
We can directly keep it as `required` instead of `validation.required`, as I don't think we need any more validations for the select field.
Thanks!